src/renderer/widgets/UnlockModal/default/model/unlockForm.ts

```diff
@@ -0,0 +1,353 @@
+import { BN, BN_ZERO } from '@polkadot/util';
+import { combine, createEvent, restore, sample } from 'effector';
+
+import { type ClaimChunkWithAccountId } from '@/shared/api/governance';
+import { type Asset, type Chain } from '@/shared/core';
+import { type Form, createForm } from '@/shared/forms';
+import { ZERO_BALANCE, getNativeAsset, nonNullable, nullable, transferableAmount } from '@/shared/lib/utils';
+import { createComplexTxStore, createSignatoriesStore } from '@/shared/transactions';
+import { type AnyAccount, accounts } from '@/domains/network';
+import { balanceModel, balanceUtils } from '@/entities/balance';
+import { locksService } from '@/entities/governance';
+import { networkModel, networkUtils } from '@/entities/network';
+import { transactionBuilder } from '@/entities/transaction';
+import { accountUtils, walletModel, walletUtils } from '@/entities/wallet';
+import { walletSelect } from '@/aggregates/wallet-select';
+import { networkSelectorModel } from '@/features/governance';
+import { locksModel } from '@/features/governance/model/locks';
+import { unlockModel } from '@/features/governance/model/unlock/unlock';
```

Comment:
can it be shortened?
```suggestion
import { unlockModel } from '@/features/governance/model/unlock';
```